devdoc/constbuffer_array_requirements.md

```diff
@@ -167,6 +169,34 @@ MOCKABLE_FUNCTION(, CONSTBUFFER_ARRAY_HANDLE, constbuffer_array_create_from_arra
 
 **SRS_CONSTBUFFER_ARRAY_42_008: [** If there are any failures then `constbuffer_array_create_from_array_array` shall fail and return `NULL`. **]**
 
+### constbuffer_array_remove_empty_buffers
```

Comment:
>constbuffer_array_remove_empty_buffers [](http://example.com/codeflow?start=4&length=38)

Should we have a function to tell us if there is an empty array in a buffer, since this function will always create a new array, it would be nice to know if we need to call this or not.